client/src/components/PrivateRoute/index.tsx

```diff
@@ -21,6 +21,9 @@ const PrivateRoute: FC<RouteProps> = ({ ...routeProps }) => {
           {isAuthenticated && !isRegistered && (
             <Redirect to="/join" from={location.pathname} />
           )}
+          {isAuthenticated && isRegistered && !user.hasRoleApproved && (
+            <Redirect to="/resources" from={location.pathname} />
```

Comment:
So this actually created a really tiny bug I don't want you guys worrying about fixing right now. Basically, when a user finishes onboarding there is a completion page on the `/join` pathname that says thanks for onboarding. At this point the user is authenticated and is registered and does not have a role approved so it redirects to the resources page and skips passed this onboarding page. I think this will just be best fixed down the line be either changing this to a popup in a Swal or a separate page.